src/Configuration/EmbeddedListViewConfigPass.php

```diff
@@ -56,11 +56,11 @@ private function processSortingConfig(array $backendConfig)
                 $backendConfig['entities'][$entityName]['embeddedList']['sort'] = $entityConfig['list']['sort'];
             } elseif (isset($entityConfig['embeddedList']['sort'])) {
```

Comment:
Why `isset` and `strtoupper` do not require explicit namespace Same for array_key_exists, etc

Reply:
isset cant be write like this